Tools/AP_Bootloader/board_types.txt

```diff
@@ -367,6 +367,8 @@ AP_HW_ZeroOne_GNSS                   5602
 
 # IDs 6600-6699 reserved for Eagle Eye Drones
 
+# IDs 6900-6999 reserved for Easy Aerial
```

Comment:
Does Easy-Aerial need 100 IDs?

If we allocate 100 IDs to every block allocation we can only have 90 allocations in here.

We've been too liberal, really.

Reply:
ok, lets do it in blocks of 10